scripts/postinstall.sh

```diff
@@ -29,25 +29,27 @@ fi
 
 # Set up the environment.
 if [ -e .env ]; then
-
-source .env
+  source .env
+
+  function rewrite_xcode_configs {
+    VALUE=$( cat .env | { grep "$1" || true; } )
```

Comment:
```suggestion
    VALUE=$( cat .env | { grep "$1" || false; } )
```

this should be `true` if failed to look for this?

Reply:
The case here is that `grep` has result code `1` if it doesn't find what it was looking for. Due to this scripts' settings it will halt execution immediately. Solution is to force it to be true and handle empty string below in the `if`.

Due to this logic I'd leave it as it is, but maybe I am approaching this in a wrong way.